src/sys/unix/selector/epoll.rs

```diff
@@ -212,6 +217,7 @@ pub mod event {
     }
 }
 
+#[cfg(target_os = "android")]
```

Comment:
Any reason this was added? I think this test should still be okay for other targets.

Reply:
Since it's only on android where that define was missing?  Using `O_CLOEXEC` in the place of `EPOLL_CLOEXEC` works on Linux, but asserting that they be equal elsewhere doesn't seem valuable unless that substitution is necessary.  In the case of illumos, the two are not equal.

Reply:
Okay makes sense. This test seems a little odd in that if we can compare it with `EPOLL_CLOEXEC`, we might as well use it in `Selector::new`, but I don't think it needs to be removed or anything as part of this.

Reply:
@kleimkuhler I asked for this, see https://github.com/tokio-rs/mio/pull/1295#discussion_r408866646.